### ISSUE TITLE
Failed to add reference. The package 'Microsoft.Azure.Management.AppService.Fluent' tried to add a framework reference to 'System.Xml.XDocument' which was not found in the GAC. This is possibly a bug in the package. Please contact the package owners for assistance.

### ISSUE THREAD

**Issue by samuelkilada** [Azure/azure-sdk-for-net#3222]
I can't seem to install "Microsoft.Azure.Management.Fluent" because the installation of "Microsoft.Azure.Management.AppService.Fluent" always fails with this error:

> Failed to add reference. The package 'Microsoft.Azure.Management.AppService.Fluent' tried to add a framework reference to 'System.Xml.XDocument' which was not found in the GAC. This is possibly a bug in the package. Please contact the package owners for assistance.

I am targeting .NET 4.5.2

**Comment by dwaijam** [comment]
Facing the same issue. Any resolution for this yet?

**Comment by samuelkilada** [comment]
I wrestled with this so much. I was finally able to resolve it (kind of). This was so annoying.

It seemed to me to work if I started a new project in Visual Studio, but I couldn't add the package in my existing project. I therefore had to add a separate project to which I could then install the package. I could then reference my secondary project from the other project. This means you'll have to do your Azure API calls in the second project, but at least it works! Hoping for a real solution sometime soon.

**Comment by jamesosb-msft** [comment]
I'm hitting this issue to.  Unfortunately, adding the reference to a different project is not an option for me.  I need to modify our existing projects to use the APIs in this package.

**Comment by Gilad-Eisenberger** [comment]
Encountered this as well.

I tried manually installing the System.Xml.XDocument nuget to get over this issue, unfortunately it wasn't enough to resolve the issue. After installing the nuget, I had to manually add a reference to the DLL in my project from the nuget's lib\netstandard1.3 folder. 

After doing that, the nuget installs properly. Not sure if it'll run properly this way. I guess this is a workaround until there's an official solution.
